src/DetailsView/reports/components/report-sections/details-section.tsx

```diff
@@ -2,12 +2,12 @@
 // Licensed under the MIT License.
 import { css } from '@uifabric/utilities';
 import * as React from 'react';
-
 import { NewTabLink } from '../../../../common/components/new-tab-link';
 import { CommentIcon } from '../../../../common/icons/comment-icon';
 import { DateIcon } from '../../../../common/icons/date-icon';
 import { UrlIcon } from '../../../../common/icons/url-icon';
 import { NamedSFC } from '../../../../common/react/named-sfc';
+import { detailsSectionList, icon as iconClass, scanDetailsSection, screenReaderOnly } from './details-section.scss';
```

Comment:
This doesn't work on the report. For now, we need to use the "hardcoded" class names.